docs/source/getting_started/installation.rst

```diff
@@ -70,33 +75,6 @@ inside of `setup.py` with one torch version while during runtime, the version of
 (which is possible because LMCache intentionally has an unpinned torch version in `requirements/common.txt`), causing 
 linker issues that will show up as undefined symbol references.
 
-.. code-block:: bash
```

Comment:
I don't think we need to delete the installation from source ?

maybe we still keep below 

```
uv pip install vllm torch
```

```
git clone https://github.com/LMCache/LMCache.git
cd LMCache
uv pip install -e . --no-build-isolation
```